### DISCUSSION TITLE
How to decompile Assembly-CSharp.dll?

### DISCUSSION THREAD

**Discussion by koopa990** [icsharpcode/ILSpy#3188] [Help]
I have ILSpy installed. Do I drag and drop the dll on one of the .CS file?

**Comment by niravx23** [comment]
Run ILSPY.exe .On the top left corner click open and browse any c# binary(.exe or dll) to view different modules simply expand the node by clicking '+' icon.

why is this in issue btw. lol

**Comment by koopa990** [comment]
> Run ILSPY.exe .On the top left corner click open and browse any c# binary(.exe or dll) to view different modules simply expand the node by clicking '+' icon.
> 
> why is this in issue btw. lol

The sharp assemble.dll I am viewing has little to no information
![image](https://github.com/icsharpcode/ILSpy/assets/63154730/c94af626-a1df-4570-b7a1-4a6a645fc4ef)

**Comment by dgrunwald** [comment]
That's a screenshot of Visual Studio. If you want to use ILSpy you need to start ILSpy; it's a separate program.

**Comment by koopa990** [comment]
> That's a screenshot of Visual Studio. If you want to use ILSpy you need to start ILSpy; it's a separate program.

You mean starting ILspy by .exe? I dont see the .exe program. Unless I somehow downloaded and installed things incorrectly.

**Reply by niravx23** [reply]
just download and extract this [zip](https://github.com/icsharpcode/ILSpy/releases/download/v9.0-preview1/ILSpy_selfcontained_9.0.0.7618-preview1-x64.zip). You will find 'ILSPY.exe' in it .

**Reply by dgrunwald** [reply]
What exactly did you download and install?

**Reply by koopa990** [reply]
> What exactly did you download and install?

I followed the how to Build section for windows..

**Comment by koopa990** [comment]
> just download and extract this [zip](https://github.com/icsharpcode/ILSpy/releases/download/v9.0-preview1/ILSpy_selfcontained_9.0.0.7618-preview1-x64.zip). You will find 'ILSPY.exe' in it .

I got it!
I am currently looking for the source of the file that contains character dialogue texts. Any hint or clues on where to start?

**Reply by niravx23** [reply]
what do you mean by character dialogue?

**Reply by koopa990** [reply]
The game displaying character dialogue/texts to show they are talking, like bubble speeches
